index.html

```diff
@@ -1976,18 +2002,19 @@ <h5 id="Psionic Traits">Psionic Traits</h5>
                </td>
             </tr>
             <tr>
-               <td>Modify Memory
+               <td>Occupy
                </td>
                <td>
-                  You may modify any 10 minutes of memory from a creature from the last week. The change in memory lasts
-                  until their next long rest - at which point they do not realise they have been
-                  affected. 2 WILL points to resist. If they successfully resist the effect the target creature will be 
-                  aware something tried to control their thoughts, but will not know the source unless you cast the 
-                  ability right in front of them or they are very knowledgeable in psionic magic.
+                  You may touch a creature to occupy it's head. You see through it's eyes and can communicate with it via thoughts.
+                  While this is happening are and are <a class="condition" data-tooltip-container>immobilised</a> and only slightly 
+                  aware of your surrondings.  You can influence this creature, giving it either adv (+) or dis (-) on each of its skill 
+                  checks.  You can end this at any time and it automatically ends if you take damage.
```

Comment:
Sounds cool, no range limit is good here I reckon, but should probably make it a max duration of 1 hour? I think we need to do more than just immobilise you while this happens - it should make you uncoscious but you do not fall prone and still very mildly aware of touch and hearing. Can you hear throught the creature's ears or is it just see and communicate? The Adv/DisAdv probs needs to go, it already has cool use case, that's just mega strong. Send one ally in on a scouting mission and give them Adv on everything.

Reply:
My thinking was adv is generally worse than having a second pair of hands / body around or splitting the party.  Particularly if you're in a place you can be discovered.

Reply:
Yes, but then you open up a whole gameplay mechanic where the party is going in somewhere to do something, maybe social or scouting, and you want to leave this guy outside and put him in the person who will be doing the talking. We'd be actively encouraging some really weird party splitting and as a GM I'm very much not ok with that. As a situational thing used in a similar way it's fine, but I don't want to be generically encouraging players to find a way to do this by slapping Adv on it.